src/include/rados/librados.hpp

```diff
@@ -10,6 +10,7 @@
 #include <utility>
 #include "memory.h"
 #include "buffer.h"
+#include "common/ceph_time.h"
```

Comment:
is this suitable for a public interface? seems like we might want to avoid exposing it